backend/alembic/versions/4ae4ff0f990a_migration.py

```diff
@@ -21,9 +21,6 @@
 
 
 def upgrade() -> None:
-    # Manually run this so that we start off with postgis enabled. It is installed with the docker image,
-    # but must be enabled manually for each database.
-    op.execute("CREATE EXTENSION IF NOT EXISTS postgis;")
```

Comment:
This command actually enables postgis. I think it's still necessary...

Reply:
No, it's done in the [docker/postgres/create-databases.sh](https://github.com/dataforgoodfr/13_brigade_coupes_rases/blob/9e17d39ffd667941ef1aa92413347dc70a5d1136/docker/postgres/create-databases.sh) script now, and I verified it works.
The new version actually solves the issue I had where this command was not executed for the test database.